### PR TITLE
Install poetry

### PR DESCRIPTION
**Pull request by freeseacher**
Poetry is supported by Pycharm by default but its installation is a bit pain. That merge request simplify it

### CODE REVIEW

**Comment by johanfleury** [comment]
Hey, thanks for the PR.

How did you generate this list of packages?

I just tried with [`flatpak-pip-generator`](https://github.com/flatpak/flatpak-builder-tools/tree/master/pip) and it seems that you’ve included some extra packages and some packages with outdated versions. Also I feel like having an external file rather than putting everything in the main file would cleaner.

Could you please try with `flatpak-pip-generator` and use include according to the tool’s README.

**Comment by freeseacher** [comment]
oh, i've done it bymyself one by one. and first part of them are from "the internets" (c)
i will regenerate package list with tool above and update pr

**Comment by freeseacher** [comment]
Switching to that tool also switched from whl dists to tar.gz and that required even more dependencies. 
and even more updates in previous stages. so i've updated them all :roll_eyes:

**Comment by freeseacher** [comment]
@johanfleury up ?

**Comment by johanfleury** [comment]
Hey, sorry, didn’t have time to review, will do this weekend or next week.

File: com.jetbrains.PyCharm-Professional.yaml [hunk 1]
```diff
@@ -20,34 +20,205 @@ modules:
...
+        sha256: c9a875ce9d7fe32887784274dd533c57909b7b1dcadcc128a2ac21331a9765dd
+      - type: file
+        url: https://files.pythonhosted.org/packages/86/c2/8fb98b8197ff6a96263eb3f39029942a412c5148c03f319c31eee8f4b03e/poetry-1.1.8.tar.gz
+        sha256: 6dc82e85d919ab99a0b18baea1933a00a40d85eedce24f80a6099909e387914e
+      - type: file
+        url: https://files.pythonhosted.org/packages/76/94/7a764d57d0f46534e0022e651da6547bc5cfe7b6372e7e0ed1dde6f5cb67/shellingham-1.4.0-py2.py3-none-any.whl
+        sha256: 536b67a0697f2e4af32ab176c00a50ac2899c5a05e0d8e2dadac8e58888283f9
+
+
+#  - name: miniconda
```
Comment (johanfleury) [inline com.jetbrains.PyCharm-Professional.yaml:214]: Can you remove these commented lines please?

**Review by johanfleury — CHANGES_REQUESTED** [review]
I don’t have time to review this in details but apart from the commented lines, it seems good. Just makes this quick change and I’ll merge it.

File: com.jetbrains.PyCharm-Professional.yaml (hunk 1 above)
Comment (freeseacher) [reply com.jetbrains.PyCharm-Professional.yaml]: yep done

**Pull request merged**